source/MessageHandlers/SendDataHandler.ts

```diff
@@ -146,9 +187,26 @@ export default class SendDataHandler implements IMessageHandler {
                             }
                         });
 
-                        await this.ingestMetadata(aggregateMetadata, folderPath);
+                        // Ingest metadata
```

Comment:
This callback's getting pretty beefy. Is it possible to move any related manifesty things out into a function?